src/utils/V3Utils.ts

```diff
@@ -88,6 +89,10 @@ export function isSlowFill(fill: Fill): boolean {
   return isV2Fill(fill) ? fill.updatableRelayData.isSlowRelay : fill.updatableRelayData.fillType === FillType.SlowFill;
 }
 
+export function isSlowFillRequest(fill: Fill | SlowFillRequest): fill is SlowFillRequest {
+  return !isV2Fill(fill as Fill) && !isV3Fill(fill as Fill);
+}
+
```

Comment:
I don't think this works. I think it will always return false. isV2Fill and isV3Fill are only meant to differentiate V2 and V3 fills from one another. Since SlowFillRequest and V3Fill share the field used to test in isV3Fill (inputToken).

Here's my way to test the type narrowing to see if ts agrees with me (I hover over the fill variable in vscode). If ts agrees with the conditions laid out above, I should see the type of fill as SlowFillRequest when used within the inner if. Instead, ts says never (meaning it thinks I can never reach this, screenshot below).
```ts
  if (!isV3Fill(fill)) {
    if (!isV2Fill(fill)) {
      fill
    }
  }
```

<img width="275" alt="Screenshot 2024-02-07 at 11 58 47 PM" src="https://github.com/across-protocol/sdk-v2/assets/11791551/467e7192-8c8e-4cf8-84d5-d3bc6c9c62df">

```suggestion
// If needed, this can be made generic in the future.
export function isSlowFillRequest(fill: Fill | SlowFillRequest): fill is SlowFillRequest {
  // Relayer field exists in V2Fill, V3Fill, but not SlowFillRequest.
  return (fill as Fill).relayer === undefined;
}
```